src/cares_wrap.cc

```diff
@@ -69,6 +69,19 @@ using v8::Value;
 
 namespace {
 
+inline uint16_t cares_get_16bit(const unsigned char* p) {
```

Comment:
@XadillaX You make a good point anyway in that I we might not be able to just assume that `buf` is 16-bit or 32-bit aligned. So, no, this should be good.